testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -292,7 +292,6 @@ public void verifyDataProviderCompleteness()
     }
 
     @Test(dataProvider = "storageFormatsWithConfiguration", groups = {STORAGE_FORMATS, HMS_ONLY})
-    @Flaky(issue = ERROR_COMMITTING_WRITE_TO_HIVE_ISSUE, match = ERROR_COMMITTING_WRITE_TO_HIVE_MATCH)
```

Comment:
have two separate commits. One which moves to `onTrino()`. The other which removes `@Flaky` annotation.

Reply:
make title of second commit slightly shorter, and put some context in the message (why the `@Flaky`  is no longer needed)